.azure-pipelines.yml

```diff
@@ -410,7 +412,7 @@ jobs:
     submodules: true
   - task: UsePythonVersion@0
     inputs:
-      versionSpec: '$(python.version)'
+      versionSpec: '$(python.version_for_macos_packaging)'
   - bash: |
```

Comment:
instead of packaging the `Parsec.app` folder in a tar.bz2, shouldn't we just do the dmg packaging here ?